pkg/prommetrics/prommetrics.go

```diff
@@ -204,3 +216,7 @@ func DecrementCRDTotal(crdType, namespace string) {
 
 	crdTotalsGaugeVec.WithLabelValues(crdType, namespace).Dec()
 }
+
+func SetBuildInfo() {
```

Comment:
```suggestion
func RecordBuildInfo() {
```

Also, could you please move this up, right after `RecordScaledObjectError()`, so somewhere around line 188. Please also add a comment with explanation to the function.